src/bentoml/_internal/service/service.py

```diff
@@ -120,6 +122,73 @@ class Service:
     _working_dir: str | None = attr.field(init=False, default=None)
     _import_str: str | None = attr.field(init=False, default=None)
 
+    __serialization_strategy: SerializationStrategy = "EXPORT_BENTO"
+
+    @property
+    def serialization_strategy(self):
+        return self.__serialization_strategy
+
+    @serialization_strategy.setter
+    def serialization_strategy(self, serialization_strategy: SerializationStrategy):
+        self.__serialization_strategy = serialization_strategy
```

Comment:
This won't work because Service is a frozen object, which `on_setattr` will raise an error when setting attributes.

Reply:
will move to a global config instead